accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+
+At any given point in time, the latest available version of all the manifests for an SDK series must always be able to be combined into a single consistent whole (in memory, not persisted to disk).
+
+Each manifest may have a `WorkloadManifest.targets` MSBuild file beside it that is unconditionally imported into all projects. This targets file contains conditioned imports to add automatic referenced to SDK packs based on project properties, and its usage and recommended patterns are described in more detail in the [workload resolvers spec](/workload-resolvers.md).
+
+## Packaging
+
+Manifests are packaged and distributed as NuGet packages. They will use the existing package type called `DotnetPlatform` so they cannot be accidentally used as a `PackageReference`. The manifest’s `sdk-manifest.json` and `WorkloadTargets.targets`  are in the root folder of the NuGet. The ID of the NuGet is `{manifest-id}-{sdk-band}` and the version of the NuGet is the manifest version.
+
+As manifests are distributed as NuGets, it’s possible to use preview versions of workloads by subscribing to preview feeds.
+
+## Installation
+
+Manifests are installed to `dotnet/sdk/{sdk-band}-manifests/{manifest-id}/` in an unpacked form. These are called the *installed manifests*.
+
+The dotnet SDK would be expected to include baseline versions of the installed manifests for all manifests that are known and supported. Although any components listed in this baseline manifest need to work with that SDK version, it does not need to be fully up-to-date: it exists primarily for listing workloads that are available to be installed, and for its ID to be known by the SDK when fetching updated versions from NuGet.
+
+Any workload management operation (install, uninstall, repair update) must be a transactional operation that also updates the installed manifests to the latest available versions for that SDK band.
+
+## Advertising
+
+The .NET tooling will automatically and opportunistically download updated versions of all manifests for the current SDK band and unpack them to `~/.dotnet/sdk-advertising/{sdk-band}/{manifest-id}/`. These user-local updated copies of the manifest are known as *advertising manifests.* The advertising manifests are used when listing workloads that are available to be installed, and for the build tooling to produce warnings that newer versions of installed components are available*.* They are **not** used in pack resolution.
+
+# Format
+
+Manifests are json files. Comments are supported, both `//` and `/* */` styles.
```

Comment:
Would be super useful to see the minimally-conformant manifest, according to these format rules. Once I see that, I'll ask what the UX or scenario would be for that minimally conformant manifest.

Reply:
This is conformant but not very useful except as an ID registration for downloading an updated manifest:
```json
{
    "version": 0,
    "workloads": {},
    "packs": {}
}
```